### ISSUE TITLE
Customizable marks

### ISSUE THREAD

**Issue by hadrysmateusz** [hanford/remark-slate#14]
It would be great if the mark types could be customized like nodes.

For example, to change `strikeThrough` to `strikethrough` or `strike-through`.

**Comment by hanford** [comment]
I agree, this seems like something we should add support for.

Probably can use the same API as default node types: https://github.com/hanford/remark-slate/blob/master/src/deserialize.ts#L39

**Comment by hadrysmateusz** [comment]
Yeah, I was thinking the same. When I have some free time I might make a PR, unless you want to do it yourself.
